explorer/src/cli/bootstrap.ts

```diff
@@ -0,0 +1,15 @@
+import { closeDbConnection, getDb } from '../database'
+
+export async function bootstrap(cb: any) {
+  const db = await getDb()
+  try {
```

Comment:
can we fit both these statements into one try/catch block instead?

Reply:
Do you have a specific reason? Closing the connection can raise an error so needs error handling.